spec/did_metadata.md

```diff
@@ -0,0 +1,112 @@
+the last event in the [[ref: KERI event stream]] that was used to construct the DID document according to
+the rules in section [DID Document from KERI Events](#did-document-from-keri-events).
+
+If the DID parameter `versionId` (see section [Support for `versionId`](#support-for-versionid)) was used when
+resolving the `did:webs` DID, and if the DID Resolution process was successful, then this corresponding DID
+document metadata property is guaranteed to be equal to the value of the DID parameter.
+
+Example:
+
+```json
+{
+  "didDocument": {
+    "id": "did:webs:example.com:Ew-o5dU5WjDrxDBK4b4HrF82_rYb6MX6xsegjq4n0Y7M"
+    // ... other properties
+  },
+  "didResolutionMetadata": {
+  },
+  "didDocumentMetadata": {
+    "versionId": "2"
+  }
+}
+```
+
+#### Use of `nextVersionId`
+
+This DID document metadata property indicates the next version of the DID document after the version that has been
+resolved.
+
+In the case of `did:webs`, this is defined to be the sequence number (i.e. the `s` field) of
+the next event in the [[ref: KERI event stream]] after the last one that was used to construct the DID document
+according to the rules in section [DID Document from KERI Events](#did-document-from-keri-events).
+
+This DID document metadata property is only present if the DID parameter `versionId`
+(see section [Support for `versionId`](#support-for-versionid)) was used when resolving the `did:webs` DID, and
+if the value of that DID parameter was not the sequence number of the last event in the [[ref: KERI event stream]].
+
+Example:
+
+```json
+{
+  "didDocument": {
+    "id": "did:webs:example.com:Ew-o5dU5WjDrxDBK4b4HrF82_rYb6MX6xsegjq4n0Y7M"
+    // ... other properties
+  },
+  "didResolutionMetadata": {
+  },
+  "didDocumentMetadata": {
+    "versionId": "1",
+    "nextVersionId": "2"
+  }
+}
+```
+
+#### Use of `equivalentId`
+
+This DID document metadata property indicates other DIDs that refer to the same subject and are logically equivalent
+to the DID that has been resolved. It is similar to the `alsoKnownAs` DID document property (see section
+[Also Known As](#also-known-as)), but it has even stronger semantics, insofar as the logical equivalence is
+guaranteed by the DID method itself.
+
+In the case of `did:webs`, this metadata property SHOULD contain a list of other known `did:webs` DIDs that differ
+in the domain name and/or port portion of the [[ref: method-specific identifier]]
+but share the same AID. Also see section [Equivalent Identifiers](#equivalent-identifiers).
+
+Example:
+
+```json
+{
+  "didDocument": {
+    "id": "did:webs:example.com:Ew-o5dU5WjDrxDBK4b4HrF82_rYb6MX6xsegjq4n0Y7M"
+    // ... other properties
+  },
+  "didResolutionMetadata": {
+  },
+  "didDocumentMetadata": {
+    "equivalentId": [
+      "did:webs:foo.com:Ew-o5dU5WjDrxDBK4b4HrF82_rYb6MX6xsegjq4n0Y7M",
+      "did:webs:bar.com:Ew-o5dU5WjDrxDBK4b4HrF82_rYb6MX6xsegjq4n0Y7M"
```

Comment:
Perhaps include a port difference example as well

Reply:
Good idea, addressed in https://github.com/trustoverip/tswg-did-method-webs-specification/pull/49/commits/5ed733235ffc2854f546fcab77375b1aef70b562